docs/wallet/wallet-extension.md

```diff
@@ -24,104 +22,133 @@ Let's walk through the steps how to install and set up the Maiar DeFi Wallet ext
 * You should receive a notification that the extension has been added successfully. 
   ![maiar-defi-install-pop-up](/wallet/wallet-extension/wallet_extension_step3.png)
 
+
 ### Set up Maiar DeFi wallet
 * Once it has been successfully installed, click on the extension to get started.
 
 * You will be presented with two options: you can either *Create new wallet* or *Import existing wallet.*
 
-
 ![maiar-defi-example-welcomepage](/wallet/wallet-extension/wallet_extension_step4.png)
 
-* We will be creating a new wallet. Click on "*Create new wallet"*. However, if you have an existing wallet, click on *Import existing wallet*. 
-* A list of secret phrases will be displayed. Copy this phrase and agree to the instruction below.
 
-> **Note**: These secret phrases should be kept confidential and not revealed to anyone.
+### Create a new wallet
+
+**Step 1:** To get started, first install the [Maiar DeFi wallet extension](https://chrome.google.com/webstore/detail/maiar-defi-wallet/dngmlblcodfobpdpecaadgfbcggfjfnm).
+
+**Step 2:** Open up the extension and click on _‘’Create new wallet”_.
+
+**Step 3:** Next, a set of 24-secret phrases will be displayed. Safely backup these secret phrases. We strongly recommend you copy the phrases or write them down and store in a safe place like a password manager.  These phrases would be used to access your wallet and may not be retrieved.
 
 ![maiar-defi-example-secretphrase](/wallet/wallet-extension/wallet_extension_step5.png)
 
-* For security purposes, you will be prompted to input some of the secret words.
+**Step 4:** Before proceeding to the next step, confirm that you have safely stored your secret phrase.
+
+**Step 5:** For further verification, you will be prompted to input some of the secret phrases. 
 
 ![maiar-defi-example-authentication](/wallet/wallet-extension/wallet_extension_step6.png)
 
-* Next, create a new password and confirm your password.
+**Step 6:** Create a password that will be used to access your wallet. Ensure you keep this password safe as it will be needed to access your wallet regularly and this password cannot be retrieved. 
```

Comment:
```suggestion
**Step 6:** Create a password that will be used to access the wallets stored in the Maiar DeFi wallet extension. Ensure you keep this password safe as it will be needed to access your wallets regularly. Please note that this password cannot be recovered if lost. 
```